Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM debian:jessie
+MAINTAINER Praekelt Foundation <dev@praekeltfoundation.org>
+
+RUN apt-get -qq update && apt-get -qqy install python-pip python-dev
+RUN pip install junebug
+RUN mkdir logs
+
+EXPOSE 8080
+ENTRYPOINT ["jb", "--config=/app/example-config.yaml"]
```

Comment:
Hmm. I'm not sure. I've got it setup to store junebug's logs in the app folder, but the channel specific logs will be stored in the container. Maybe we should have Junebug create the logging folder if it doesn't exist.